src/metrics/rules/active_ruleset_test.go

```diff
@@ -3282,6 +3291,7 @@ func TestMatchedKeepOriginal(t *testing.T) {
 			0,
 			nil,
 			rollups,
+			nil,
```

Comment:
Do we not want to add tests for UtilizationRules here?

Reply:
removing any changes related for `ActiveRuleSet` since this is not gonna be used for metric-rule matching